html/user-activation/activation-same-and-cross-origin.sub.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="resources/utils.js"></script>
+  </head>
+  <body>
+    <h1>Check that cross origin iframes don't get activated</h1>
+    <p>
+      Tests that activating a same-origin navigable doesn't activate a cross
+      origin navigable.
+    </p>
+    <ol id="instructions">
+      <li>Click inside the yellow area.</li>
+    </ol>
+    <h1>Same origin</h1>
+    <iframe id="so-child" width="200" height="50"></iframe>
+    <h1>Cross origin</h1>
+    <iframe id="xo-child" width="200" height="50"></iframe>
+  </body>
+  <script>
+    const soChild = document.getElementById("so-child");
+    const xoChild = document.getElementById("xo-child");
+
+    function receiveMessage(type) {
```

Comment:
This method will be useful for other tests too, like `navigation-state-reset-sameorigin.html`.  Please consider moving this to `resources/utils.js`.